schema/enum.go

```diff
@@ -35,6 +35,15 @@ func Parse%v(val string) %v {
 }
 `
 
+const enumTypeValidator = `
+func Validate%v(val string) bool {
```

Comment:
Maybe Rename from `Validate%v` to `Is%v`
```suggestion
func Is%v(val string) bool {
```

Reply:
Made the change throughout